models.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from flask import Flask
+
+import sqlite3
+from flask import g
+
+DATABASE = "baza.db"
+
+app = Flask(__name__)
+
+
+# na tyle ile umiałem, nie do końca jeszcze kumam z wyciągnięciem kursora z bloku,
+# nie wiem czy dobrze ale połączyło mi się udało mi się dla delete
+# update mi wyrzuca błąd konfliktu od dwóch obiektów
+# prowadź Sensei proszę ;)
+
+#pzdr
+
+
+class Todos:
+
+
+    def __init__(self):
+        self.db = sqlite3.connect(DATABASE)
+
+#w co mam opakować dalsą cześć bloku ?
+
+        self.cur = self.db.cursor()

```

Comment:
to tak jak mówiłem, możesz ten blok wynieść do nowej metody w obiekcie

Reply:
tzn. zastąpić wszystko co jest w metodzie `all` tym co teraz masz tutaj, tylko używając lokalnej zmiennej dla kursora, tzn. powinno być `cursor = self.db.cursor()` zamiast `self.cur...`

Reply:
no zwracając z metody `all` ten słownik `todos`, który wypełniłeś w pętli